mantidproject/mantid
# Fix incident energy calibration for PANTHER

**Description of work.**
See the title.

**Report to:** [ILL/Rols]

**To test:**


1. `rm -rf ~/.mantid/instrument`
2. 
```
DirectILLCollectData(Run='ILL/PANTHER/002687', OutputWorkspace='out', IncidentEnergyCalibration='Energy Calibration ON', OutputIncidentEnergyWorkspace='ei')
``` 
Make sure that 'out' has a sample log named `Ei` with value 75.37, which should also be written in `ei` single valued workspace (be careful, opening single valued workspace currently crash workbench).

The data file you will find in the unit tests data directory.

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
